docs/curl/datatx.md

```diff
@@ -1,213 +1,412 @@
 # Data Transaction
-We can store/update/delete any state on the ledger by issuing a data transaction. By submitting a `POST /data {txPaylod}`, we can perform a data transaction where `txPayload` contains reads, writes, and deletes of states.
 
-> Before executing the commands in the document, we need to create a user `user1` with read-write privilege to the database `bdb`. Refer [here](usertx.md) to add a new user `user1` with a read-write database privilege. We also need to create `user2` but no privileges are required.
+We can store, update, delete any state, i.e., key-value pair, on the ledger by issuing a data transaction.
+By submitting a `POST /data/tx {txPaylod}`, we can perform a data transaction where `txPayload` contains reads, writes, and deletes of states.
+
+> As a prerequisite, we need to create users `alice` and `bob`. Refer [here](./usertx.md) for examples on creating users. Further, the node should have a user database named `db1`.
 
 ## Storing a new state
 
-Let's store a new state with the key `key1`. 
+Let's store a new state with the key `key1`.
 
 ```json
  curl \
    -H "Content-Type: application/json" \
-   -X POST http://127.0.0.1:6001/data \
+   -H "TxTimeout: 2s" \
+   -X POST http://127.0.0.1:6001/data/tx \
    --data '{
 	"payload": {
-		"userID": "user1",
-		"txID": "1b6d6414-9b58-45d0-9723-1f31712add80",
-		"DB_name": "bdb",
-		"data_writes": [
-			{
-				"key": "key1",
-				"value": "dmFsdWUy",
-                "ACL": {
-					"read_users": {
-						"user2": true
-					},
-					"read_write_users": {
-						"user1": true
-					}
-				}
-			}
+		"must_sign_user_ids": [
+            "alice"
+        ],
+		"tx_id": "1b6d6414-9b58-45d0-9723-1f31712add81",
+        "db_operations": [
+            {
+                "db_name": "db2",
```

Comment:
`db2`? above you say `db1`?

Reply:
This happens throughout the example

Reply:
Done. `db2` is the correct usage. Needed changes only in one place in the text.